smhi/smhi/smhi_lib.py

```diff
@@ -323,6 +326,7 @@ def _get_forecast(api_result: dict) -> List[SmhiForecast]:
         forecast._temperature_min = forecast_temp_min
         forecast._total_precipitation = total_precipitation
         forecast._mean_precipitation = total_precipitation / 24
+        forecast._wind_speed = acc_wind_speed / len(forecasts_day)

```

Comment:
What do you think about we keep the _wind_speed as original and introduce a _mean_wind_speed instead? Then we do not break existing that expects 1-1 from SMHI api?

Reply:
That is the sensible way to do it, so I just commited it :)